src/flask_app.py

```diff
@@ -108,7 +108,7 @@ def restart_flask(self) -> None:
         if not self._charm_state.is_secret_storage_ready:
             logger.info("secret storage is not initialized")
             return
-        container.add_layer("flask-app", self._flask_layer(), combine=True)
+        container.add_layer("flask", self._flask_layer(), combine=True)
```

Comment:
Can we use `FLASK_SERVICE_NAME` here or the layer name can be different?

Reply:
I think the `label` argument or `"flask"` here corresponds to the layer filename in `/var/lib/pebble/default/layers` i.e. `/var/lib/pebble/default/layers/002-flask.yaml`. Therefore, I think the update here doesn't affect the service configuration, so I probably shouldn't change here.